regimes/gr/README.md

```diff
@@ -0,0 +1,134 @@
+# 🇬🇷 GOBL Greece Tax Regime
+
+Greece uses the myDATA and Peppol BIS Billing 3.0 formats for their e-invoicing/tax-reporting system.
+
+Example GR GOBL files can be found in the [`examples`](./examples) (YAML uncalculated documents) and [`examples/out`](./examples/out) (JSON calculated envelopes) subdirectories.
+
+## Public Documentation
+
+* [myDATA API Documentation v1.0.7](https://www.aade.gr/sites/default/files/2023-10/myDATA%20API%20Documentation_v1.0.7_eng.pdf)
+* [Greek Peppol BIS Billing 3.0](https://www.gsis.gr/sites/default/files/eInvoice/Instructions%20to%20B2G%20Suppliers%20and%20certified%20PEPPOL%20Providers%20for%20the%20Greek%20PEPPOL%20BIS-EN-%20v1.0.pdf)
+* [VAT Rates](https://www.gov.gr/en/sdg/taxes/vat/general/basic-vat-rates)
+
+## Greece specifics
+
+### VAT categories
+
+Greece has three VAT rates: standard, reduced and super-reduced. Each of these rates are reduced by a 30% on the islands of Leros, Lesbos, Kos, Samos and Chios. The tax authority identifies each rate with a specific VAT category.
+
+In GOBL, the IAPR VAT category code must be set using the `iapr-gr-vat-category` extension of a line's tax to one of these codes:
```

Comment:
```suggestion
In GOBL, the IAPR VAT category code must be set using the `iapr-gr-vat-cat` extension of a line's tax to one of these codes:
```

Reply:
Addressed in [512ab72](https://github.com/invopop/gobl/pull/307/commits/512ab72a85133320e6986ac3f1e59dc2fab0285c)